src/Microsoft.Kusto.ServiceLayer/LanguageServices/AutoCompleteHelper.cs

```diff
@@ -2,630 +2,117 @@
-            "session",
-            "session_user",
-            "sessionproperty",
-            "set",
-            "sets",
-            "setuser",
-            "shutdown",
-            "sign",
-            "simple",
-            "sin",
-            "size",
-            "smallint",
-            "smallmoney",
-            "snapshot",
-            "some",
-            "soundex",
-            "space",
-            "sql",
-            "sqrt",
-            "square",
-            "standard",
-            "start",
-            "started",
-            "state",
-            "statement",
-            "static",
-            "statistics",
-            "statistics_norecompute",
-            "status",
-            "stdev",
-            "stdevp",
-            "stopped",
-            "str",
-            "string_agg",
-            "stuff",
-            "substring",
-            "sum",
-            "sysdatetime",
-            "sysname",
-            "system",
-            "system_time",
-            "system_user",
-            "table",
-            "take",
-            "tan",
-            "target",
-            "temporary",
-            "then",
-            "throw",
-            "time",
-            "timestamp",
-            "tinyint",
-            "to",
-            "top",
-            "tran",
-            "transaction",
-            "translate",
-            "trigger",
-            "trim",
-            "true",
-            "truncate",
-            "try",
-            "try_convert",
-            "tsql",
-            "type",
-            "uncommitted",
-            "unicode",
-            "union",
-            "unique",
-            "uniqueidentifier",
-            "unknown",
-            "unpivot",
-            "update",
-            "updatetext",
-            "upper",
-            "use",
-            "user",
-            "user_name",
-            "using",
-            "value",
-            "values",
-            "var",
-            "varchar",
-            "varp",
-            "varying",
-            "version",
-            "view",
-            "waitfor",
-            "when",
-            "where",
-            "while",
-            "with",
-            "within",
-            "without",
-            "work",
-            "write",
-            "writetext",
-            "xact_abort",
-            "xml",
-            "year",
-            "zone",
-        };
-
-        /// <summary>
-        /// Gets a static instance of an empty completion list to avoid
-        /// unneeded memory allocations
-        /// </summary>
-        internal static CompletionItem[] EmptyCompletionList
-        {
-            get
+        public static CompletionItem[] GetDefaultAutoComplete(DataSourceType dataSourceType, ScriptDocumentInfo scriptDocumentInfo, Position textDocumentPosition){
+            switch (dataSourceType)
             {
-                return AutoCompleteHelper.emptyCompletionList;
-            }
-        }
+                case DataSourceType.Kusto:
```

Comment:
Can we move this into the DataSource folder. Maybe have a static function there with a switch for DataSourceType.